meshroom/ui/graph.py

```diff
@@ -363,8 +364,10 @@ def importProject(self, filepath):
                 localFile = filepath.toString()
         else:
             localFile = filepath
+        if isinstance(position, QPoint):
+                position = Position(position.x(), position.y())
```

Comment:
Should we not import at the center of the GraphEditor when we do not have a mouse position, like for the copy/paste nodes?